.circleci/conditional_config.yml

```diff
@@ -275,7 +283,7 @@ jobs:
 
   migrations_tests:
     docker:
-      - <<: *frontend_backend
+      - <<: *frontend_ci
```

Comment:
we don't need chromium here

Reply:
We also don't need the backend here. :+1: